components/config/date_picker.py

```diff
@@ -2,7 +2,7 @@
 import dash_html_components as html
 
 
-def get_component(min_date, max_date, default_end_date):
+def get_component(min_date, max_date, default_end_date, id):
```

Comment:
For the date-picker to still work elsewhere, the `id` parameter should have the default value `date-pick`

Reply:
Done!